src/aurora/fevfile.h

```diff
@@ -45,6 +47,14 @@ class FEVFile {
 		kStreamFromDisk
 	};
 
+	/** Some objects in FMOD can have generic properties. */
+	struct Property {
+		uint32 type;
+		int32 integerValue;
+		float floatValue;
+		Common::UString stringValue;
```

Comment:
Hmm, that same pattern, a variable with multiple potential types, comes up again and again in the codebase now. In the ActionScript code, the NWScript code, the Lua code, etc.

Maybe be should see if we can use Boost.Any or Boost.Variant for this instead? This code might be the ideal test environment for that. Could you see if you can do that?

For the differences between Any and Variant, see https://www.boost.org/doc/libs/1_53_0/doc/html/variant/misc.html#variant.versus-any . I'd say Variant sounds like the better candidate, with Any a fall-back if Variant should not work for our use case.

You probably still need variable to store the current type, though. I'd rather that be an enum instead of just an integer without any obvious meaning, too.